clouddriver-aws/src/main/groovy/com/netflix/spinnaker/clouddriver/aws/deploy/ops/DeleteAmazonSnapshotAtomicOperation.java

```diff
@@ -0,0 +1,57 @@
+package com.netflix.spinnaker.clouddriver.aws.deploy.ops;/*
```

Comment:
weird spot for a header line ya got there... is this a long con to steal @robzienert 's thunder if he ever tries to rename packages?

Reply:
Intellij and I are no longer friends. It's trying to sabotage me